db/fixtures/miq_product_features.yml

```diff
@@ -1822,263 +1822,206 @@
   :identifier: rss
 
 # Policy
-- :name: Explorer
-  :description: Control Explorer
+- :name: Control
+  :description: Control
   :feature_type: node
-  :identifier: control_explorer
+  :identifier: control
   :children:
-  - :name: View All Records
-    :description: View all Records in Control Explorer
-    :feature_type: view
-    :hidden: true
-    :identifier: control_explorer_view
```

Comment:
@h-kataria Okay, this is one is actually used by the API, and am not sure if there is a equivalent permission available.

Previous question ref: https://github.com/ManageIQ/manageiq/pull/20438/files#r525530422
API Config in question: https://github.com/ManageIQ/manageiq-api/blob/32b9f8e2f53ed85fcfd16cae1aa07d03fae18b51/config/api.yml#L2442-L2453
cc @gtanzillo

Reply:
@NickLaMuro `control_explorer_view` feature no longer exists, it has been replaced by just a parent feature `control`, it looks like we no longer need the block in question in api.yml //cc @gtanzillo

Reply:
@h-kataria but that is the thing: This isn't simply removing the block, but it is completely removing controller actions/routes by doing so.

If I expand the the chunk of code I am referencing a bit further:

https://github.com/ManageIQ/manageiq-api/blob/32b9f8e2f53ed85fcfd16cae1aa07d03fae18b51/config/api.yml#L2435-L2453

You will see that we would be removing basically all of the `:policy_actions:` controller routes by just "removing it".  If that is the intent, okay, but it probably isn't something that we should just remove unceremoniously without some kind of deprecation period.  If we _**don't**_ plan on removing these actions, then there needs to be an equivalent permission available for those actions so these routes can continue to function.

Reply:
There are matching features, I guess we can patch this https://github.com/ManageIQ/manageiq-api/pull/966